flutter_module/lib/core/color.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+const MaterialColor green = MaterialColor(_greenColor, <int, Color>{
+  50: Color(0xFFE0F0EF),
+  100: Color(0xFFB3DAD6),
+  200: Color(0xFF80C2BB),
+  300: Color(0xFF4DAAA0),
+  400: Color(0xFF26978B),
+  500: Color(_greenColor),
+  600: Color(0xFF007D6F),
+  700: Color(0xFF007264),
+  800: Color(0xFF00685A),
+  900: Color(0xFF005547),
+});
+const int _greenColor = 0xFF008577;
+
+const MaterialColor pink = MaterialColor(_pinkColor, <int, Color>{
+  50: Color(0xFFFAE4EC),
+  100: Color(0xFFF3BBCF),
+  200: Color(0xFFEC8DB0),
+  300: Color(0xFFE45F90),
+  400: Color(0xFFDE3D78),
+  500: Color(_pinkColor),
+  600: Color(0xFFD41858),
+  700: Color(0xFFCE144E),
+  800: Color(0xFFC81044),
+  900: Color(0xFFBF0833),
+});
+const int _pinkColor = 0xFFD81B60;
```

Comment:
Ideally, this should be wrapped in a class and use it as a namespace to prevent unnecessary issues.

Reply:
It is now wrapped in `AppColor`